llvm/lib/Target/AArch64/AArch64FrameLowering.cpp

```diff
@@ -3703,6 +3703,17 @@ bool isMergeableStackTaggingInstruction(MachineInstr &MI, int64_t &Offset,
   return true;
 }
 
+bool isNZCVLiveOut(MachineBasicBlock &MBB) {
```

Comment:
I'm wondering why this need a new function to be implemented.
Wouldn't calling `MBB.isLiveOut(MBB, AArch64::NZCV)` work the same as calling `isNZCVLiveOut(MBB)`?

Reply:
I can not find `MBB.isLiveOut` anywhere. Does it exist? There is LiveVariables::isLiveOut, but that only works for virtual regs.

Anyway, I've not touched backend code in a long while. This seems very conservative but correct to me.